verifact/forum/schema.py

```diff
@@ -39,3 +47,93 @@ def resolve_questions(root, info):
 
     def resolve_answers(root, info):
         return Answer.objects.all()
+
+
+class QuestionCreate(Mutation):
```

Comment:
I'm sorry, I should have caught this in the first run. This should actually inherit from `graphene.relay.ClientIDMutation` to comply with the relay spec. Relay has specific things that should be present in requests, including a clientId. It also requires that input be specified as a single `input` object that contains the fields in question.